### PR TITLE
pl-PL: bring up to date and improve

### PR DESCRIPTION
**Pull request by arekdurlik**
Applying for issue(s):
- #2414
- #2430
- #2446
- #2466
- #2472
- #2473
- #2492
- #2501
- #2509
- #2511
- #2520
- #2527
- #2535
- #2550
- #2551
- #2562
- #2563
- #2572
- #2578
- #2594
- #2602
- #2609
- #2620
- #2625
- #2631
- #2641
- #2650
- #2658
- #2659
- #2678
- #2679
- #2692
- #2720
- #2732
- #2755
- #2769
- #2781
- #2794
- #2801
- #2806
- #2814
- #2815
- #2821
- #2826
- #2836
- #2843
- #2854
- #2868
- #2869
- #2875
- #2881
- #2884
- #2894
- #2895
- #2897
- #2909
- #2917
- #2928
- #2936
- #2938
- #2941
- #2945
- #2960

### CODE REVIEW

File: data/language/pl-PL.txt [hunk 1]
```diff
@@ -1113,8 +1119,8 @@ STR_1726    :Teren nie jest na sprzedaż!
 STR_1727    :Prawa do budowy nie są na sprzedaż!
 STR_1728    :Nie można tu kupić praw do budowy…
 STR_1729    :Teren nie należy do parku!
-STR_1730    :{RED}Zamknięte - - 
-STR_1731    :{WHITE}{STRINGID} - - 
+STR_1730    :{RED}Zamknięte
+STR_1731    :{WHITE}{STRINGID} - -
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:1123]: This needs a trailing space.

File: data/language/pl-PL.txt [hunk 2]
```diff
@@ -1155,7 +1161,7 @@ STR_1771    :{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{COM
 STR_1773    :Tylko jedna sekcja fotograficzna dostępna na atrakcję
 STR_1774    :Tylko jedna wyciągarka dostępna na atrakcję
 STR_1777    :{WINDOW_COLOUR_2}Muzyka atrakcji
-STR_1778    :{STRINGID} - - 
+STR_1778    :{STRINGID} - -
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:1164]: This one too.

File: data/language/pl-PL.txt [hunk 3]
```diff
@@ -720,18 +726,18 @@ STR_1331    :{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{VEL
...
 STR_1336    :{STRINGID} - Wjazd {POP16}{COMMA16} do stacji
-STR_1337    :{STRINGID} - Opuszczanie{POP16}{POP16}
+STR_1337    :{STRINGID} - Wyjście{POP16}{POP16}
 STR_1338    :{STRINGID} - Opuszczanie {POP16}{COMMA16} stacji
 STR_1339    :{BLACK}Oczekiwanie na rezultaty testu…
 STR_1340    :{WINDOW_COLOUR_2}Maks. prędkość: {BLACK}{VELOCITY}
 STR_1341    :{WINDOW_COLOUR_2}Czas przejazdu: {BLACK}{STRINGID}{STRINGID}{STRINGID}{STRINGID}
 STR_1342    :{DURATION}
-STR_1343    :{DURATION} / 
+STR_1343    :{DURATION} /
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:737]: This one too.

File: data/language/pl-PL.txt [hunk 4]
```diff
@@ -720,18 +726,18 @@ STR_1331    :{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{VEL
...
 STR_1344    :{WINDOW_COLOUR_2}Czas przejazdu: {BLACK}{STRINGID}{STRINGID}{STRINGID}{STRINGID}
 STR_1345    :{LENGTH}
-STR_1346    :{LENGTH} / 
+STR_1346    :{LENGTH} /
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:740]: This one too.

File: data/language/pl-PL.txt [hunk 5]
```diff
@@ -1977,10 +1983,10 @@ STR_2779    :Podgląd #{COMMA16}
 STR_2780    :Dodatkowe okno podglądu
 # Koniec nowych treści
 STR_2781    :{STRINGID}:
-STR_2782    :SHIFT + 
-STR_2783    :CTRL + 
+STR_2782    :SHIFT +
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:1986]: These two as well.

File: data/language/pl-PL.txt [hunk 6]
```diff
@@ -2780,21 +2786,21 @@ STR_5616    :Ostatni fragment flagi
...
 STR_5626    :Inne parki
 STR_5627    :Grupuj scenariusze według:
 STR_5628    :Gra Źródłowa
 STR_5629    :Poziom trudności
 STR_5630    :Włącz odblokowywanie scenariuszy
 STR_5631    :Oryginalne parki z dodatku
 STR_5632    :Zbuduj własny…
-STR_5633    :CMD + 
-STR_5634    :OPTION + 
+STR_5633    :CMD +
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:2802]: Idem

File: data/language/pl-PL.txt [hunk 7]
```diff
@@ -2176,23 +2182,23 @@ STR_3140    :Wyciągarka musi znajdować się na początku trasy
 STR_3141    :Wielokrotne pętle nie są dostępne w parze z wyciągarką
 STR_3142    :{WINDOW_COLOUR_2}Pojemność: {BLACK}{STRINGID}
 STR_3143    :Pokaż ludzi
-STR_3144    :Pokaż przejażdżki i stoiska
+STR_3144    :Pokaż atrakcje i stoiska
 STR_3160    :Wybierz liczbę obrotów na przejażdzkę
 STR_3162    :Nie można zaalokować odpowiedniej ilości pamięci
-STR_3163    :Instalowanie nowych danych: 
+STR_3163    :Instalowanie nowych danych:
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:2188]: This also needs trailing whitespace (sorry, missed it the first time).

File: data/language/pl-PL.txt [hunk 8]
```diff
@@ -2176,23 +2182,23 @@ STR_3140    :Wyciągarka musi znajdować się na początku trasy
...
 STR_3164    :{BLACK}{COMMA16} wybrano (maksymalnie {COMMA16})
 STR_3167    :{WINDOW_COLOUR_2}Zawiera: {BLACK}{COMMA16} obiektów
-STR_3169    :Nie znaleziono danych dla obiektu: 
+STR_3169    :Nie znaleziono danych dla obiektu:
```
Comment (Gymnasiast) [inline data/language/pl-PL.txt:2191]: This one too.

**Comment by Gymnasiast** [comment]
Thanks! Would you like to be notified of new strings from now on? If so, I’ll put you on the list.

**Comment by arekdurlik** [comment]
Sure, I'll be happy to help!

**Pull request merged**